compiler/optimizer/VPHandlers.cpp

```diff
@@ -12223,12 +12280,23 @@ TR::Node *constrainArrayStoreChk(OMR::ValuePropagation *vp, TR::Node *node)
       TR::VPConstraint *object = vp->getConstraint(objectRef, isGlobal);
       TR::VPConstraint *array  = vp->getConstraint(arrayRef, isGlobal);
 
-      // If the object reference is null we can remove this check.
+      const bool areValueTypesEnabled = TR::Compiler->om.areValueTypesEnabled();
```

Comment:
Is this dead?

Reply:
Oops!  Yes.  It was from an earlier version of these changes.

Reply:
Removed in revised commit https://github.com/eclipse/omr/pull/5203/commits/e5dd6f4fa2e8ef9eef41d432a706cce2111b5e15